src/config/yaml_baking_conf_parser.py

```diff
@@ -156,8 +154,14 @@ def validate_baking_address(self, conf_obj):
 
         # key_name must has a length of 36 and starts with tz or KT, an alias is not expected
         if len(baking_address) == PKH_LENGHT:
-            if not baking_address.startswith("tz"):
-                raise ConfigurationException("Baking address must be a valid tz address")
+            if not baking_address.startswith("tz") and not baking_address.startswith("KT"):
```

Comment:
I added the KT address which can also pass because of the comment mentioned above. Is this fine?

Reply:
I am not sure about this. Is a KT address delagatable?

Reply:
Found a [resource](https://crypto.bi/tz-kt/): Originated addresses (TK) cannot act as bakers. Only manager addresses (tz) can be registered to bake.

Reply:
This is a case that supports Robert Cecil Martin who claims "comments lie!" ;)